portfolio/src/main/webapp/education.html

```diff
@@ -8,28 +8,75 @@
+      <li><a href="mixes.html">Music Mixes</a></li>
     </ul>
 
+    <h2><u>Education</u></h2>
+      <span><img src="images/pclogo.png" 
+                 width=150px height=175px alt="Pine Crest Logo"></span>
+        <p><i><b>Pine Crest School</b></i> - Pine Crest School is a
+          private prepatory school in Fort Lauderdale, Florida.
+          I attended from August 2015 to May 2019.</p>
 
-    <h4><u>Education</u></h4>
-    <p><i>Pine Crest School</i> - Pine Crest School is a private prepatory
-     school in Fort Lauderdale, Florida. I attended from August 2015 to 
-     May 2019.</p>
-    <p><i>Columbia University</i> - I currently attend Columbia University
-     in the City of New York with an anticipated major in Computer Science.
-    My expecyted graduation date is May 2023.</p>
+      <span><img src="images/Columbia_Engineering_logo.png" 
+                 width=150px 
+                 height=175px 
+                 alt="Columbia University Crown">
+        <p><i><b>Columbia University</b></i> - I currently attend Columbia 
+          University in the City of New York with an anticipated major in 
+          Computer Science. My expected graduation date is May 2023.</p>
 
-    <h4><u>Activities</u></h4>
\ No newline at end of file
+    <h2><u>Activities</u></h2>
+      <h5>At Columbia I partake in a few clubs and organizations,
+        I have listed most of them below:</h5>
+        <span><img src="images/Bacch B.png" 
+                   width=150px 
+                   height=175px 
+                   alt="Bacchanal Logo"></span>
+          <p><i><b>Bacchanal</b></i> - I am a member of Columbia University's
+            largest music event organization, Bacchanal. I am the tech chair 
+            on the e-board hat organizes the event and throws events under
+            the organizations name.</p>
+            
+        <span><img src="images/NSBELogo.png" 
+                   width=150px 
+                   height=175px 
+                   alt="NSBE Logo"></span>
+        <p><i><b>The National Society of Black Engineers</b></i>
+          - I am the acting telecommunications chair for Columbia's NSBE 
+          chapter.NSBE allows me to connect with fellow African-American
+          engineers within my college community as well as sharing studying
+          resources and a safe place to express myself.</p></br>
+
+        <span><img src="images/readahead.jpg" 
+                   height=150px 
+                   width=350px 
+                   alt="NSBE Logo"></span>
+        <p><i><b>Read Ahead</b></i> - I am a mentor with Read Ahead, 
+          an organization that connects adults and college students with
+          underpriviledged elementary and middle school kids in Morningside
+          Heights.</p>
+
+        To download my full resume, please click the icon
+          <a href="images/resume.pdf" download> 
+            <img src="images/download icon.png" alt="Resume Download"
```

Comment:
Ha, you missed formatting this one.

Reply:
Slightly confused on the formatting I missed? Can you elaborate?